src/main/java/org/gridsuite/voltageinit/server/VoltageInitController.java

```diff
@@ -112,4 +112,14 @@ public ResponseEntity<Void> stop(@Parameter(description = "Result UUID") @PathVa
         voltageInitService.stop(resultUuid, receiver);
         return ResponseEntity.ok().build();
     }
+
+    @GetMapping(value = "/results/{resultUuid}/modifications-group", produces = APPLICATION_JSON_VALUE)
```

Comment:
here we get the modifications group uuid not a resource, maybe we should do something for the path ?

Reply:
Changed into /results/{resultUuid}/modifications-group-uuid